src/main/java/org/gridsuite/filter/server/utils/expertfilter/ExpertFilterUtils.java

```diff
@@ -128,6 +146,20 @@ private static String getTerminalFieldValue(FieldType field, Terminal terminal)
         };
     }
 
+    private static String getSectionBasedFieldValue(FieldType field, ShuntCompensator shuntCompensator) {
+        double susceptancePerSection = shuntCompensator.getModel(ShuntCompensatorLinearModel.class).getBPerSection();
+        double qAtNominalV = Math.pow(shuntCompensator.getTerminal().getVoltageLevel().getNominalV(), 2) * Math.abs(susceptancePerSection);
+
+        return switch (field) {
+            case SHUNT_COMPENSATOR_TYPE -> susceptancePerSection > 0 ? "CAPACITOR" : "REACTOR";
+            case MAX_Q_AT_NOMINAL_V -> String.valueOf(qAtNominalV * shuntCompensator.getMaximumSectionCount());
+            case SWITCHED_ON_Q_AT_NOMINAL_V -> String.valueOf(qAtNominalV * shuntCompensator.getSectionCount());
+            case MAX_SUSCEPTANCE -> String.valueOf(susceptancePerSection * shuntCompensator.getMaximumSectionCount());
+            case SWITCHED_ON_MAX_SUSCEPTANCE -> String.valueOf(susceptancePerSection * shuntCompensator.getSectionCount());
```

Comment:
Why `SWITCHED_ON_MAX_SUSCEPTANCE` and not `SWITCHED_ON_SUSCEPTANCE` ?

Reply:
the key in grid study is "SwitchedOnMaxSusceptance", so I used it like that

Reply:
`    "SwitchedOnMaxSusceptance": "Switch-on susceptance",`
Looking at the translation, I think it's a mistake.